gaphor/diagram/diagramlabel.py

```diff
@@ -4,3 +4,8 @@
 @singledispatch
 def diagram_label(diagram):
     return diagram.name
+
+
+@singledispatch
+def paint_diagram_type(diagram) -> bool:
+    return True
```

Comment:
```suggestion
    return not diagram.diagramType
```

Reply:
discarded `not` because we are checking whether we should paint the diagram type.